e2e_tests/test_standalone/test_study_list.py

```diff
@@ -25,31 +28,49 @@ def test_home(
     assert title == "Optuna Dashboard (Wasm ver.)"
 
 
+def create_rdb_storage_sqlite_file(filename: str, study_name: str):
+    storage = optuna.storages.RDBStorage(f"sqlite:///{filename}")
+    study = optuna.create_study(study_name=study_name, storage=storage)
+
+    def objective(trial: optuna.Trial) -> float:
+        x1 = trial.suggest_float("x1", 0, 10)
+        x2 = trial.suggest_float("x2", 0, 10)
+        return (x1 - 2) ** 2 + (x2 - 5) ** 2
+
+    study.optimize(objective, n_trials=100)
+
+
+def create_journal_storage_local_file(filename: str, study_name: str):
+    storage = optuna.storages.JournalStorage(
+        optuna.storages.JournalFileStorage(f"{filename}"),
+    )
+    study = optuna.create_study(study_name=study_name, storage=storage)
+
+    def objective(trial: optuna.Trial) -> float:
+        x1 = trial.suggest_float("x1", 0, 10)
+        x2 = trial.suggest_float("x2", 0, 10)
+        return (x1 - 2) ** 2 + (x2 - 5) ** 2
+
+    study.optimize(objective, n_trials=100)
+
+
+@pytest.mark.parametrize(
+    "create_storage_file",
+    [create_rdb_storage_sqlite_file, create_journal_storage_local_file],
+)
```

Comment:
[suggestion] How about bundling them together?
```suggestion
def create_rdb_storage_file(filename: str, study_name: str, storage_type: str):
    if storage_type == "rdb":
        storage = optuna.storages.RDBStorage(f"sqlite:///{filename}")
    elif storage_type == "journal":
        storage = optuna.storages.JournalStorage(
            optuna.storages.JournalFileStorage(f"{filename}"),
        )
    else:
        assert False, f"Got an unexpected storage_type={storage_type}."

    study = optuna.create_study(study_name=study_name, storage=storage)

    def objective(trial: optuna.Trial) -> float:
        x1 = trial.suggest_float("x1", 0, 10)
        x2 = trial.suggest_float("x2", 0, 10)
        return (x1 - 2) ** 2 + (x2 - 5) ** 2

    study.optimize(objective, n_trials=100)


@pytest.mark.parametrize("storage_type", ["rdb", "journal"])
```